frontend/src/components/homepage/Filter.tsx

```diff
@@ -40,43 +51,49 @@ const Filter = ({
       <option key={i.toString()} value={i.toString()} label={i.toString()} />,
     );
   }
+  const filterStyle = useStyleConfig("Filter");
   return (
-    <div className="filter">
-      <h1>Filter Stories</h1>
-      <div>
-        <h4>TRANSLATION LANGUAGE</h4>
-        <select
+    <Flex sx={filterStyle}>
```

Comment:
Not sure we should be using the `sx` prop like this, generally think it's bad practice in ChakraUI and whatever can be included in a theme should be.